demisto_sdk/commands/lint/helpers.py

```diff
@@ -281,18 +281,26 @@ def add_tmp_lint_files(content_repo: git.Repo, pack_path: Path, lint_files: List
 
 
 @lru_cache(maxsize=100)
-def get_python_version_from_image(image: str) -> float:
+def get_python_version_from_image(image: str, timeout: int = 60) -> float:
     """ Get python version from docker image
 
     Args:
         image(str): Docker image id or name
+        timeout(int): Docker client request timeout
 
     Returns:
         float: Python version X.Y (3.7, 3.6, ..)
     """
+
+    excluded_images = {'demisto/powershell', 'demisto/powershell-ubuntu', 'demisto/pwsh-exchange',
+                       'demisto/pwsh-infocyte', 'demisto/powershell-teams'}
```

Comment:
maybe it is best to check if the name includes `powershell`/`pwsh` instead of a fixed set of images (in case we add a new PowerShell-based image, it will require a new SDK version.

Reply:
thought about it, decided against it for the possibility of excluding images that happen to have `pwsh` in the name - what do you say?